README.md

```diff
@@ -16,15 +18,25 @@ To install, follow the [installation instructions for your Operating System](htt
 
 1. Get local copies of the project submodules so you can build and run your site locally:
    - `git submodule update --init --recursive`
-
-2. Build your site:
+1. Build your site:
    - `hugo server`
    - If there are any errors, check that you have the latest version of Hugo, and are using the extended version.
+1. Preview your site in your browser at: http://localhost:1313/
+1. As you make changes to the site, your browser will automatically reload your local dev site so you can easily and quickly see your changes. 
 
-3. Preview your site in your browser at: http://localhost:1313/
+## Optional Link Checking
 
-## Continuous Deployment
+We validate that all links on the docs site work (do not 404) using a tool called [Muffet](https://github.com/raviqqe/muffet) along with  [Actions](https://github.com/features/actions).  If you're creating a lot of new links, or editing a lot of existing links, you may optionally run Muffet locally before pushing your commits.  This will save a lot of time of pushing a build, waiting for the Action to run, finding you have a broken link, fixing it, pushing a new change etc.
```

Comment:
Not sure if the double spaces were typos, a typing preference, or both :)

We generally have single spaces after periods so have made that correction in this suggestion. Also, the last sentence was confusing to read at first... tried to make it clearer, but could use yet another pass:

```suggestion
We validate that all links on the docs site work (do not 404) using a tool called [Muffet](https://github.com/raviqqe/muffet) along with [Actions](https://github.com/features/actions). If you're creating a lot of new links, or editing a lot of existing links, you may optionally run Muffet locally before pushing your commits. Running Muffet locally can save time by exposing broken links before pushing a build since you can avoid waiting for the Action to run, finding you have a broken link, fixing it, and pushing a new change.
```

Reply:
I'd say this old dog is having trouble learning new tricks ;) My typing class 30 years ago (on actual typewriters!!) has me doing double spaces after periods.  The other spaces...are just mistakes?

Either way, thanks for the space fixes.  And thanks for the re-wording - agreed this is clearer what's going on.